app/models/editionable_worldwide_organisation.rb

```diff
@@ -19,6 +19,39 @@ def display_type_key
     "editionable_worldwide_organisation"
   end
 
+  alias_method :original_main_office, :main_office
+
+  # I'm trying to use a domain centric design rather than a persistence
+  # centric design, so I do not want to expose a has_many :home_page_lists
+  # and all that this implies. I really only want to expose a list of
+  # contacts (in order) that should be shown on the home page, and some
+  # simple (explicit) methods for manipulating them.
```

Comment:
It's probably not worth copying this comment over, since this code has existed for a long time and I'm not convinced it adds any additional information.